guides/swoosh_mailer.md

```diff
@@ -22,8 +24,15 @@ defmodule MyAppWeb.PowMailer do
 
   def process(email) do
     deliver(email)
+    |> log_warnings()
   end
+
+  defp log_warnings({:error, reason}) do
+    Logger.warn("Mailer backend failed with: #{inspect(reason)}")
+  end
+
+  defp log_warnings(ok), do: ok
```

Comment:
Maybe this should be `defp log_warnings({:ok, email}), do: {:ok, email}`. I prefer to do this to make sure I catch successful transactions rather than just anything, in case something changes in the future.

Reply:
That's a good idea, I wasn't sure if you would want the rigidity, so I wrote it to be flexible. I do prefer your suggestion as you say; it will prevent future unknown issues.